packages/mux-player/src/index.ts

```diff
@@ -141,6 +142,9 @@ class MuxPlayerElement extends VideoApiElement {
     this.#setUpCaptionsButton();
     this.#setUpAirplayButton();
     this.#setUpVolumeRange();
+
+    this.#userInactive = this.shadowRoot?.querySelector('media-controller')?.hasAttribute('user-inactive') || true;
```

Comment:
does the condition need `??` instead of `||`. right now it will always be true

Reply:
I tried to, but the build was complaining about it. I can try again.